doc/docs/configuration.md

```diff
@@ -14,17 +14,32 @@ Serial Terminal settings are explained here. Use the default settings unless the
 
+| Terminal mode  | Setting            | Description                                                                      |
+| -------------- | ------------------ | -------------------------------------------------------------------------------- |
+| Line           | **Clear on Send**      | When **Clear on Send** is enabled, input text is cleared from the terminal input window after it has been transmitted. |
+| Line           | **Line Ending**        | Use this to send an optional line ending to the command sent to the connected device. The alternatives are:</br> - `None`</br>- `LF` sends the newline character.</br> - ``CR`` for carriage return.</br> - ``CRLF`` sends carriage return and newline.     |
+
 ### Shell mode
 Use Shell mode when the device you communicate with is running a shell, such as Zephyr™ shell. In this mode, the terminal sends the command or command series to the device for execution by the shell. The shell can return output or prompt the user for additional input. Shell mode allows you to execute more complex operations, navigate the file system, or perform other advanced tasks that are not possible in Line mode.
 
 | Terminal mode | Setting            | Description                                                                      |
-| -------------- | ------------------ | -------------------------------------------------------------------------------- |
-| Line           | **Clear on Send**      | When **Clear on Send** is enabled, input text is cleared from the terminal input window after it has been transmitted. |
-| Line           | **Line Ending**        | Use this to send an optional line ending to the command sent to the connected device. The alternatives are:</br> - `None`</br>- `LF` sends the newline character</br> - **CR** for carriage return</br> - **CRLF** sends carriage return and newline     |
-| Shell         | **Device Controls Echo** | Toggle this on if echo is enabled on the device, and toggle off if not, in which case Serial Terminal echoes the command. In Zephyr shell, run `shell echo on` to turn echo on, and `shell echo off` to disable echo. |
\ No newline at end of file
+| ------------- | ------------------ | -------------------------------------------------------------------------------- |
+| Shell         | **Device controls echo** | Toggle this on if echo is enabled on the device. Toggle off if it is not, in which case Serial Terminal echoes the command. In Zephyr shell, run `shell echo on` to turn echo on, and `shell echo off` to disable echo. |
+
+## Write to file
+
+Use the **Save to file** button to save the history of the terminal console to a file.
+
+Until saved, the console output is saved in the buffer and a temporary history file that you can configure in the **Terminal Settings** tab.
+
+## Terminal settings tab
+
+In this tab, you can configure scrollback buffer for the console and the size of the temporary history file, in which the console output is kept until you **Save to file**.
```

Comment:
What is the _temporary_ history file?

The history file is persistent, meaning it will be available in and across sessions.

The XTerm Scrollback buffers are "_voltatile_", meaning they are not persisted between or across sessions, and they are limited by the scrollback option, which is default ~1000 lines.